resources/views/partials/search-collection.blade.php

```diff
@@ -1 +1,15 @@
-<h2>Search the BC Open Textbook Collection</h2>
+@php
+    $link = site_url() . '/find-open-textbooks';
```

Comment:
Do we need to consider using `network_site_url()` instead? 
Docs suggested usage: `$link = site_url( '/find-open-textbooks', 'https' );`

Reply:
Consider using `get_home_url()` instead, see [this](https://discourse.roots.io/t/site-url-returning-wp/12270).

Reply:
nice find! `get_home_url()` will work better...`network_site_url()` will return something we don't want on prod